src/reusedBrowser.ts

```diff
@@ -345,10 +336,7 @@ export class ReusedBrowser implements vscodeTypes.Disposable {
     });
   }
 
-  private async _createFileForNewTest(model: TestModel) {
-    const project = model.enabledProjects()[0];
-    if (!project)
-      return;
+  async createFileForNewTest(model: TestModel, project: TestProject) {
```

Comment:
I think this function does not belong here anymore and should be moved to `extension.ts`. It has nothing to do with reused browser. Can be done in a follow up.